internal/sessiontest/main_test.go

```diff
@@ -213,11 +211,16 @@ func sessionHelper(t *testing.T, request irma.SessionRequest, sessiontype string
 		require.NoError(t, result.Err)
 	}
 
-	var resJwt string
-	err = irma.NewHTTPTransport("http://localhost:48682/session/"+sesPkg.Token, false).Get("result-jwt", &resJwt)
+	resultEndpoint := "result"
+	if authEnabled {
+		resultEndpoint = "result-jwt"
+	}
```

Comment:
Requestor authentication is unrelated as a feature to JWT signing of the session result. Why does the one enable the other here?

Reply:
it's a hacky solution where we make use of the coincidence that all tests that require JWT signing also test requestor authentication. i added a comment to make this clear.